src/web/browser/startup.ts

```diff
@@ -4,7 +4,7 @@ export interface Reporter {
     report: (err: Error, tags: { [key: string]: string }) => void;
 }
 
-const measure = (name: string, task: () => Promise<void>): void => {
+const measure = (name: string, task: () => Promise<any>): void => {
```

Comment:
I think this can go back to `Promise<void>` since we now shouldn't have changed the interface `measure` has?

Reply:
Good catch, thanks! Sorted now.